sympy/parsing/latex/lark/latex_parser.py

```diff
@@ -3,18 +3,23 @@
 import re
 
 from sympy.external import import_module
-from .transformer import TransformToSymPyExpr
+from sympy.parsing.latex.lark.transformer import TransformToSymPyExpr
 
 _lark = import_module("lark")
 
+
 class LarkLatexParser:
     def __init__(self, logger=False, print_debug_output=False, transform=True):
+        grammar_dir_path = os.path.join(os.path.dirname(__file__), "grammar/")
```

Comment:
"grammar" better than "grammar/", Microsoft Windows uses backslashes instead of slashes.

Reply:
Windows supports forward slashes on paths. It has done so for a long time. It's not a big deal. See [here](https://stackoverflow.com/a/12086287/12591388) for some relevant information.

FWIW, I have been using Windows as my dev environment this whole time, and it never complained.